src/pages/ProductDetail/components/Detail/Detail.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import './Detail.scss';
+import DetailCount from './DetailCount';
+import DetailHeader from './DetailHeader';
+import Navigate from './Navigate';
+import TotalCost from './TotalCost';
+const Detail = () => {
+  const [count, setCount] = useState(1);
+
+  const decrease = () => {
+    if (count > 1) {
+      setCount(count - 1);
+    }
+  };
```

Comment:
```suggestion
  const decrease = () => {
    if (count > 1) {
      setCount(prev=>prev-1);
    }
  };
```
함수형 업데이트로 이전 state를 참조해서 현재 state로 업데이트 하는 방식을 활용 해 보세요.

Reply:
반영했습니다